govcd/sample_govcd_test_config.json

```diff
@@ -29,14 +29,15 @@
 		"externalIp": "10.150.10.10",
 		"internalIp": "192.168.1.10",
     "disk": {
-      "size": 1024,
-      "sizeForUpdate": 10240
+      "size": 1048576,
```

Comment:
Comment what this 1048576 means would be good. That is Megabyte in bytes.

Reply:
The descriptions are only in the .yaml file